lld/test/ELF/pack-dyn-relocs-ifunc-static.s

```diff
@@ -0,0 +1,26 @@
+# REQUIRES: aarch64
```

Comment:
This file can be merged into lld/test/ELF/pack-dyn-relocs-ifunc.s by adding another asm file c.s

Reply:
Done